NDTensors/src/imports.jl

```diff
@@ -5,6 +5,10 @@
 # https://github.com/oschulz/HeterogeneousComputing.jl
 using .Expose: cpu
 
+import .CUDAExtensions: cu
+import .GPUArraysCoreExtensions: storagemode
+import .MetalExtensions: mtl
```

Comment:
Let's not import these, and overload them with the syntax `CUDAExtensions.cu(...) = ...` as needed.

Reply:
Okay I removed these imports. I only did this so that `mtl` and `cu` could live in `NDTensors` namespace, see the comment in `CUDAExtensions`